pom.xml

```diff
@@ -109,6 +103,46 @@
 						</execution>
 					</executions>
 				</plugin>
+				
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-javadoc-plugin</artifactId>
+					<executions>
```

Comment:
You've deleted `<version>3.6.3</version>` in the first pom, but didn't add it here, so it falls back to an older not pinned version. 
Please add this line here: `<version>3.6.3</version>`